frontend/packages/akr/src/components/clerkTranslator/overview/AuthorisationListing.tsx

```diff
@@ -76,16 +75,14 @@ export const AuthorisationListing = ({
             <TableCell>{t('fields.endDate')}</TableCell>
             <TableCell>{t('fields.permissionToPublish')}</TableCell>
             <TableCell>{t('fields.diaryNumber')}</TableCell>
-            <TableCell>{translateCommon('delete')}</TableCell>
+            <TableCell>{t('fields.actions')}</TableCell>
           </TableRow>
         </TableHead>
         <TableBody>
           {authorisations.map((a, i) => (
             <TableRow
               key={a.id ?? a.tempId}
-              data-testid={`authorisations-table__id-${
-                a.id ?? `${i}-unsaved`
-              }-row`}
+              data-testid={`authorisations-table__id-${a.id ?? `${i}`}-row`}
```

Comment:
Tää `-unsaved` oli aikoinaan tarpeellinen koska auktorisointien id:t voi periaatteessa olla samoja kuin silmukan taulukon indeksit => vältetään Reactin duplicate keyt, jos taulukossa on sekä tallennettuja että tallentamattomia auktorisointeja. Oliko tän tarve varmasti poistunut?

Reply:
Tallentamattomia ja tallennettuja auktorisointeja auktorisointien listauksessa ei pitäis kyllä samaan aikaan voida olla. Voihan tätä tosin pyöritellä vielä varmistukseksi.

Reply:
Joo, en kyllä itsekään saanut toistettua tilannetta jossa tää ois ollu tarpeellinen - ehkä se oli relevantti ennen tätä PR:ää?

Reply:
Mietin lähinnä itse että oliko pikemminkin havainnollistamiseksi tehty tuo että data-testid:t erilaisia kääntäjän tarkastelusivulla vs. lisäyssivulla. Ei tosiaan tullut mietittyä tätä sen enempää kun tuota muutin.

Reply:
Muistan kyllä siis että tälle on aikoinaan ollut ihan tarve ettei cypress-testit mee sekaisin. Mutta tarve on tosiaan hyvin mahdollisesti poistunut jo jonkun aikaa sitten..